rudder-server-root/SPECS/rudder-server-root.spec

```diff
@@ -52,7 +52,7 @@ Source8: rudder-server-root
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch: noarch
 
-Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, rudder-agent, curl
+Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, Rudder::Agent, curl
```

Comment:
It will resolve it as a dependency, and either install the the regular or their thin client. Not sure if you can have a weight which to choose.